### ISSUE TITLE
site facelift/revamp/update ideas

### ISSUE THREAD

**Issue by jgravois** [Esri/esri.github.io#36]
a little :bird: told me our friendly neighborhood DC R&D has some good ideas for a facelift :japanese_goblin: for the esri github landing page.

> It's time to improve the landing experience to really speak legitimately what we have and how it can be used. Devs new to ArcGIS - what is the scope and capability of different projects. What's useful without any license vs what is really an extension as a customer. How do I get started.

care to elaborate @ajturner?

SAP uses a custom [DESCRIPTION.MD](https://github.com/cla-assistant/cla-assistant/blob/master/DESCRIPTION.MD) in repositories to hydrate the cards on [their landing page](http://sap.github.io)...

**Comment by ajturner** [comment]
:dart: 

We have a broad variety of open source projects: from fully-functional apps, down to low-level geometry libraries. It can be a bit overwhelming to understand the landscape of tools, libraries, apps, and components. 

I think we should provide a _guide_ to visitors that first lays out _why_ Esri thinks open-source is so important and a vital part of our technology, platform, business and community. We can then outline the different types of projects we have and how they might be useful to developers.

For example [NetFlix Github](http://netflix.github.io/) has narrative sections that give context to how the pieces fit together. 

For us that may be Solution Applications, Geoprocessing Tools, Infrastructure Operations, Design, Customer Support...  These can be curated and not have _all_ projects, just the curated ones that really exemplify good practices and general value.

Then we can still have a catalog, and I like SAP example because it provides a summary description before the person needs to click through to the README. 

Lastly, would it interesting to consider if the page could be built using a StoryMap (since they are open-source) but that may be a bit weird.

**Comment by alaframboise** [comment]
Ya, it's definitely time we revisit this.

**Comment by ajturner** [comment]
Here is a list of suggested sections. These are not comprehensive, but curated guide of projects 

- Visualization
 - ESRI-Leaflet, Cedar, Wind-JS, D3 layer
- Analysis
 - R Bridge, GIS Tools Hadoop
- Applications
- Mobile
- Desktop
 - OSM Editor
- Data Processing
 - Terraformer, Koop, Geomertry Engine
 - API libraries in Python, Ruby, JavaScript
 - Geoportal
- Ops
 - PDX tools
- General Support
 - GeoDev and other support projects

Any others?

**Comment by jgravois** [comment]
a good example of articulating [*why*](https://github.com/uscensusbureau/open-source-policy) (and maybe even *how*).

**Comment by ajturner** [comment]
@jgravois @patrickarlt @phpmaps  Here is a mock based on our whiteboard session for an updated landing page that curates projects. It is only suggestions of categories & repos per category.

/cc  @alaframboise @nixta 

![esri open source homepage](https://cloud.githubusercontent.com/assets/1218/12985312/de6cc9dc-d0bf-11e5-8ebf-841bb43ee9d7.png)

**Comment by jgravois** [comment]
@ajturner this is *super* helpful.  reminder to myself that using calcite-web and acetate would be a good idea.

**Comment by jgravois** [comment]
looks like we got beat to the punch
http://www.esri.com/software/open/open-source

**Comment by ajturner** [comment]
No, that is a different message and list of projects. It doesn't provide categories of projects, case studies, and will be long cycles to update.

I helped with those pages and had that in mind when we discussed these designs the other week. Those marketing pages should link to the deeper more technical pages like esri.github.io and developers.

**Comment by jgravois** [comment]
making progress...
https://jgravois.github.io/esri.github.com/

**Comment by jf990** [comment]
I took a crack at this seeing how we are linking to this site from the home page on [developers](http://developers.arcgis.com) but the content here hasn't been updated in a rather long time. Also based on a ad-hoc discussion @jgravois and I had over lunch:

 * refactored to use Acetate, Calcite-web, node/npm
 * the query for github projects is now a live query against github/Esri instead of hard-coded projects on the page. this will allow the page to catch new projects and accurately report forks/stars/last update. TODO: how to exclude specific projects.
 * includes the curated section of projects driven from a local data file and static at build time. this allows us to change these projects without editing html pages.

if we decide this project has merit, there's still a bit of work to do, such as figuring out what are the curated projects. some help with design. fixing up the search selector but I kept the one that was already there. Also, since this is built to a separate folder we need to figure out how to stage this on github.
